src/styles/card.scss

```diff
@@ -0,0 +1,109 @@
+$heightProduct: 402px;
+$widthProduct: 200px;
+$borderRadius: 5px;
+$padding: 14px;
+
+.product {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: $borderRadius;
+  height: $heightProduct;
+  width: $widthProduct;
+
+  &__photo {
+    height: 134px;
+    width: 160px;
+    margin-left: 19px;
+    margin-top: 30px;
+  }
+
+  &__card {
+    padding-left: $padding;
+    padding-right: $padding;
+    margin-top: 45px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+    color: #000;
+  }
+
+  &__code {
+    margin-top: 5px;
+    font-weight: normal;
+    @include sizeFont;
+    color: #000;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__rating-stars {
+    display: flex;
+    margin-left: 1px;
+    margin-right: 1px;
+    margin-top: 1px;
```

Comment:
![image](https://user-images.githubusercontent.com/33497431/87339952-12b30880-c550-11ea-87ab-da1c222724c9.png)

Reply:
remember this?